ext/app/electron/main.ts

```diff
@@ -1,39 +1,27 @@
 import * as dotenv from "dotenv";
-import {commonUrls} from 'app/common/gristUrls';
 import * as version from 'app/common/version';
 import * as log from 'app/server/lib/log';
 import * as electron from 'electron';
 import * as fse from 'fs-extra';
-import * as os from 'os';
 import * as path from 'path';
 import * as winston from 'winston';
-
-dotenv.config();
-
-// Handle --version flag, which causes us to only print version, without running anything.
-if (process.argv.includes('--version')) {
-  console.log(`${version.version} (${version.gitcommit} on ${version.channel})`);
-  process.exit(0);
+import { loadConfigFile } from "app/electron/config";
+import { program } from 'commander'
+import * as packageJson from 'desktop.package.json';
```

Comment:
try to match surrounding style with new imports, although I see dotenv contributor already scrambled it a bit.

Reply:
Are you suggesting to always avoid importing names directly from modules (and rather always import whole modules)?

Reply:
No. Just linty things like the kind of quote marks in use, and spacing, and alphabetical order. We don't have a global convention, the convention is to follow local convention.